src/ZipFile.jl

```diff
@@ -287,20 +290,48 @@ function _find_enddiroffset(io::IO)
     offset
 end
 
+_find_enddiroffset(io::IO) = _find_sigoffset(io, _EndCentralDirSig)
+_find_zip64_enddirlocoffset(io::IO) = _find_sigoffset(io, _Zip64EndCentralLocSig)
+
+function _find_zip64_enddiroffset(io::IO, locoffset::Integer)
+	seek(io, locoffset)
+	if readle(io, UInt32) != _Zip64EndCentralLocSig
+		error("internal error")
+	end
+	skip(io, 4)
+	readle(io, UInt64) #the offset of the z64ecd
+end
+function _find_zip64_diroffset(io::IO, enddiroffset::Integer)
+	seek(io, enddiroffset)
+	if readle(io, UInt32) != _Zip64EndCentralDirSig
+		error("internal error")
+	end
+	skip(io, 8+2+2+4+4+8)
+	nfiles = readle(io, UInt64)
+	skip(io, 8) #skip size of central directory
+	offset = readle(io, UInt64)
+	offset, nfiles
+end
 function _find_diroffset(io::IO, enddiroffset::Integer)
-    seek(io, enddiroffset)
-    if readle(io, UInt32) != _EndCentralDirSig
-        error("internal error")
-    end
-    skip(io, 2+2+2)
-    nfiles = read(io, UInt16)
-    skip(io, 4)
-    offset = readle(io, UInt32)
-    commentlen = readle(io, UInt16)
+	seek(io, enddiroffset)
+	if readle(io, UInt32) != _EndCentralDirSig
+		error("internal error")
+	end
+	skip(io, 2+2+2)
+	nfiles = read(io, UInt16)
+	skip(io, 4)
+	offset = readle(io, UInt32)
+	commentlen = readle(io, UInt16)
```

Comment:
these lines shouldn't be indented

Reply:
My bad! My text editor somehow inserted tabs instead of 4 spaces. Fixed in 3648030cfc5e91d58f11559cc23e446cb197223e and 4538bf2f8bdc1c72d534bcff454ed09223ead863.